Solutions/ImpervaCloudWAF/Data Connectors/ImpervaWAFCloud_FunctionApp.json

```diff
@@ -0,0 +1,121 @@
+{
+	"id": "ImpervaWAFCloudAPI",
+	"title": "Imperva Cloud WAF",
+	"publisher": "Atlassian",
+	"descriptionMarkdown": "The [Imperva Cloud WAF](https://www.imperva.com/resources/resource-library/datasheets/imperva-cloud-waf/) data connector provides the capability integrate and to ingest Web Aplication Firewall events into Azure Sentinel through the REST API. Refer to Log integration [documentation](https://docs.imperva.com/bundle/cloud-application-security/page/settings/log-integration.htm#Download) for more information. The connector provides ability to get events which helps to examine potential security risks, analyze your team's use of collaboration, diagnose configuration problems and more.",
+	"additionalRequirementBanner": "These queries and workbooks are dependent on a parser based on Kusto to work as expected. Follow the steps to use this Kusto functions alias **ImpervaWAFCloud** in queries and workbooks [Follow steps to get this Kusto functions>](https://aka.ms/sentinel-impervawafcloud-parser).",
```

Comment:
Please update additionalRequirementBanner as follows.
`">This data connector depends on a parser based on a Kusto Function to work as expected [**ImpervaWAFCloud**](https://aka.ms/sentinel-impervawafcloud-parser) which is deployed with the Azure Sentinel Solution."`

Reply:
done